requirements/docs-requirements.txt

```diff
@@ -220,29 +225,58 @@ gevent==21.8.0
     #   django-websocket-redis
 ghdiff==0.4
     # via -r base-requirements.in
-google-api-core==2.5.0
-    # via google-api-python-client
+google-api-core[grpc]==2.11.0
```

Comment:
we are upgrading quite a few existing libraries here. We might need to cross check on features that are using these libraries, to make sure they work okay with newer version.